libraries/JGUZDV.JobHost/src/JGUZDV.JobHost/src/RegisterHost.cs

```diff
@@ -20,7 +20,7 @@ internal class RegisterHost : IJob, IHostedService
         private readonly IOptions<JobReportOptions> _options;
 
         public RegisterHost(IJobExecutionManager reporter,
-            IEnumerable<RegisterJob> jobs,
+            IEnumerable<RegisterJob> jobs, // TODO: RegisterJobs need to be injected in a different way since they are scoped but the RehisterHost is a singleton
```

Comment:
?

Reply:
Das war nur ein Kommentar bis ich dazu einen PR mache der das behebt. (Siehe PR "Getting list of jobs from ServiceProvider")